tools/deployment/presto-clp/docker-compose.yaml

```diff
@@ -50,5 +50,6 @@ volumes:
   worker-config:
 
 networks:
-  presto:
-    driver: "bridge"
+  clp-package:
+    name: "${CLP_PACKAGE_NETWORK_NAME:-clp-package_default}"
```

Comment:
Why not make it required? Or do you think the default would be useful in some way?

Reply:
the default is useful when users don't use `start_clp.py` to set up the Package compose project (so instance IDs won't be appended to the project name), and when they don't use the `init.py` script to set up the Presto compose project

which is honestly not recommended to general users...

That said, at least the `clp-package` part will be consistent with the default Package compose project name:
https://github.com/y-scope/clp/blob/2ac456ec619b6f21ebdc14af0a4ff0eeba1048a5/tools/deployment/package/docker-compose.yaml#L1